src/collective/texttospeech/upgrades/v5/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from collective.texttospeech.logger import logger
+from plone import api
+
+
+NEW_JS = '//code.responsivevoice.org/1.5/responsivevoice.js'
+OLD_JS = '//code.responsivevoice.org/1.4/responsivevoice.js'
+
+
+def pin_responsivevoice(setup_tool):
```

Comment:
seems this upgrade step is not supported under Plone 5.1; better ask @thet for advice on how to fix this.

Reply:
@thet any advice on how to fix JS registration on Plone 5.1?

Reply:
take a look at https://github.com/collective/collective.lazysizes/pull/36 but I think is better to provide different profiles for different Plone versions.